### DISCUSSION TITLE
Doors disappearing

### DISCUSSION THREAD

**Discussion by GinoX27** [Pryaxis/TShock#1790] [Q&A] [answered]
Yo,

I am having an issue where if you abuse opening/closing a door, it will eventually disappear. But it doesn't seem to affect admins users that are logged in.

Is there a way to fix this?

**Comment by hakusaro** [comment, marked as answer]
This is a known issue: #1774 — we have to fix them, sadly. Until that issue is closed, you can give users the permission `tshock.ignore.sendtilesquare` but doing that will result in disabling a critical anticheat feature. If you trust your players, you can do that. If not, don't.
